### PR TITLE
run rustfmt on librustc_driver folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
@nikomatsakis  LGTM. What do you think?

File: src/librustc_driver/pretty.rs [hunk 1]
```diff
@@ -235,14 +235,11 @@ impl PpSourceMode {
...
-                    };
+                    let annotation = TypedAnnotation { tcx: tcx };
                     let _ignore = tcx.dep_graph.in_ignore();
-                    f(&annotation,
-                      payload,
-                      ast_map.forest.krate())
-                }), sess)
+                    f(&annotation, payload, ast_map.forest.krate())
+                }),
+                             sess)
```
Comment (jseyfried) [inline src/librustc_driver/pretty.rs]: I find this formatting confusing and hard to read.

File: src/librustc_driver/pretty.rs [hunk 2]
```diff
@@ -354,7 +353,8 @@ impl<'ast> pprust::PpAnn for IdentifiedAnnotation<'ast> {
     }
     fn post(&self, s: &mut pprust::State, node: pprust::AnnNode) -> io::Result<()> {
         match node {
-            pprust::NodeIdent(_) | pprust::NodeName(_) => Ok(()),
+            pprust::NodeIdent(_) |
+            pprust::NodeName(_) => Ok(()),
```
Comment (jseyfried) [inline src/librustc_driver/pretty.rs]: Needless added line break.

File: src/librustc_driver/pretty.rs [hunk 3]
```diff
@@ -658,11 +657,15 @@ impl fold::Folder for ReplaceBodyWithLoop {
     fn fold_block(&mut self, b: P<ast::Block>) -> P<ast::Block> {
         fn expr_to_block(rules: ast::BlockCheckMode, e: Option<P<ast::Expr>>) -> P<ast::Block> {
             P(ast::Block {
-                stmts: e.map(|e| ast::Stmt {
-                    id: ast::DUMMY_NODE_ID,
-                    span: e.span,
-                    node: ast::StmtKind::Expr(e),
-                }).into_iter().collect(),
+                stmts: e.map(|e| {
+                        ast::Stmt {
```
Comment (jseyfried) [inline src/librustc_driver/pretty.rs]: Strange indent.

File: src/librustc_driver/pretty.rs [hunk 4]
```diff
@@ -799,17 +802,18 @@ pub fn print_after_parsing(sess: &Session,
...
-            pprust::print_crate(sess.codemap(),
-                                sess.diagnostic(),
-                                krate,
-                                src_name.to_string(),
-                                &mut rdr,
-                                out,
-                                annotation.pp_ann(),
-                                false)
-        }).unwrap()
+                debug!("pretty printing source code {:?}", s);
```
Comment (jseyfried) [inline src/librustc_driver/pretty.rs]: Strange indent.

File: src/librustc_driver/pretty.rs [hunk 5]
```diff
@@ -843,82 +854,82 @@ pub fn print_after_hir_lowering<'tcx, 'a: 'tcx>(sess: &'a Session,
...
-                                    sess.diagnostic(),
-                                    krate,
-                                    src_name.to_string(),
-                                    &mut rdr,
-                                    out,
-                                    annotation.pp_ann(),
-                                    true)
-            })
-        }
+            (PpmSource(s), _) => {
```
Comment (jseyfried) [inline src/librustc_driver/pretty.rs]: Incorrect indent (looks like a `rustfmt` bug).

File: src/librustc_driver/test.rs [hunk 6]
```diff
@@ -114,9 +117,15 @@ fn test_env<F>(source_string: &str,
...
-        ).expect("phase 2 aborted")
+        driver::phase_2_configure_and_expand(&sess,
+                                             &cstore,
+                                             krate,
+                                             None,
+                                             "test",
+                                             None,
+                                             MakeGlobMap::No,
+                                             |_| Ok(()))
+            .expect("phase 2 aborted")
```
Comment (jseyfried) [inline src/librustc_driver/test.rs]: I prefer the original formatting here.

File: src/librustc_driver/pretty.rs [hunk 7]
```diff
@@ -1017,7 +1028,9 @@ fn print_with_analysis<'tcx, 'a: 'tcx>(sess: &'a Session,
             }
             _ => unreachable!(),
         }
-    }), sess).unwrap();
+    }),
+                 sess)
+        .unwrap();
```
Comment (jseyfried) [inline src/librustc_driver/pretty.rs]: This is very strange/confusing.

**Comment by srinivasreddy** [comment]
@nikomatsakis  rollup pls

File: src/librustc_driver/pretty.rs (hunk 5 above)
Comment (nrc) [inline src/librustc_driver/pretty.rs]: https://github.com/rust-lang-nursery/rustfmt/issues/1197

**Comment by nrc** [comment]
I filed an issue for the over-indented match arm. I think all the other issues are already known.

**Pull request merged**